src/DetailsView/reports/automated-checks-report.scss

```diff
@@ -64,6 +64,40 @@ $outcome-not-applicable-summary-color: $neutral-outcome;
             }
         }
     }
+
+    .rule-more-resources {
+        display: flex;
+        flex-direction: column;
+
+        margin-left: 24px;
+        margin-bottom: 16px;
+        padding: 14px 20px;
+
+        background-color: $always-white;
+
+        box-shadow: 0px 0.3px 0.9px rgba(0, 0, 0, 0.108), 0px 1.6px 3.6px rgba(0, 0, 0, 0.132);
```

Comment:
The shadow for the resources card is smaller than the shadow for the instance cards; is that expected?

Reply:
Got it from figma. @cheade can you help us with this question?

Reply:
Yep - I had the shadow on this changed slightly as it's not an interactive card like our details cards are, so I added a small variation.

Reply:
Thanks for the reply @cheade